src/popup.js

```diff
@@ -0,0 +1,90 @@
+import { getData } from './api.js';
+// eslint-disable-next-line import/no-cycle
```

Comment:
*[OPTIONAL]*


- Hey, I see that you're having an import cycle issue, I understand how this would have an effect on your codebase if you want to fix them, you'd have to refactor all the code in your codebase. however, kindly remember that linters are there to help us write clean and maintainable code. For this reason, I'd overlook this part of the linter disabling.